test/e2e/windows/memory_limits.go

```diff
@@ -105,31 +103,34 @@ func overrideAllocatableMemoryTest(f *framework.Framework, allocatablePods int)
 		podType = "memory_limit_test_pod"
 	)
 
-	totalAllocatable := getTotalAllocatableMemory(f)
+	var pod v1.Pod
+	var failurePod v1.Pod
 
-	memValue := totalAllocatable.Value()
-	memPerPod := memValue / int64(allocatablePods)
-	ginkgo.By(fmt.Sprintf("Deploying %d pods with mem limit %v, then one additional pod", allocatablePods, memPerPod))
+	selector := labels.Set{"kubernetes.io/os": "windows"}.AsSelector()
+	nodeList, err := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{
+		LabelSelector: selector.String(),
+	})
+	framework.ExpectNoError(err)
 
-	// these should all work
-	pods := newMemLimitTestPods(allocatablePods, imageutils.GetPauseImageName(), podType, strconv.FormatInt(memPerPod, 10))
-	f.PodClient().CreateBatch(pods)
+	for range nodeList.Items {
 
-	failurePods := newMemLimitTestPods(1, imageutils.GetPauseImageName(), podType, strconv.FormatInt(memPerPod, 10))
-	f.PodClient().Create(failurePods[0])
+		f.PodClient().Create(&pod)
```

Comment:
The pod spec hasn't been configured here.  The struct needs to be filled out with the basic info like: pod image, node selector (using current name of node currently being processed in the loop) and the memory to use for each pod (will come from a given Node allocatable info for current node in loop).